win32/config.vc

```diff
@@ -960,6 +961,7 @@ perl5=''
 perl='perl'
 perl_patchlevel='~PERL_PATCHLEVEL~'
 perl_static_inline='static __inline'
+perl_thread_local=''
```

Comment:
```suggestion
perl_thread_local='__declspec(thread)'
```

Reply:
I don't think that the code that I changed in `thread.h` is reachable for Win32. The Win32 code seems to be in `win32thread.h` (and already contains some conditional code to support `__declspec(thread)` that doesn't seem to be used.

Given that

1. I really don't know Win32
2. I don't have any access to Win32
3. I *think* that these code paths are all independent

it would seem better to me to leave these WIn32 config files as the default "disable", even if it's actually wrong, and then address the Win32 config in a separate PR.

Particularly given that the "fun" I already found (thank you Apple) on macOS.

So yes, agree this needs addressing, but can we do it next?

Reply:
Yeah, I guess I can prepare a PR after this one is merged (and actually test it this time :P)

Reply:
That would be cool

None of this stuff is obvious - I had a head start on you because i've been looking at it for longer, and I was forced to start by reading the entire files (rather than just looking at the changeset). And asking myself things like "What *is* `pthread_unchecked_getspecific_np` and does it matter?" (Answers seem to be "no good idea" and "probably not")

Reply:
So I think that this makes this "conversation" and the previous 3 "resolved"